lib/kitchen/verifier/busser.rb

```diff
@@ -46,7 +46,7 @@ class Busser < Kitchen::Verifier::Base
         if verifier.windows_os?
           "$env:systemdrive\\opscode\\chef\\embedded\\bin"
         else
-          "/opt/chef/embedded/bin"
+          verifier.remote_path_join(%W[#{verifier[:chef_omnibus_root]} embedded bin])
```

Comment:
I thought there was some magic where we could set it for both... but i'll wait for @fnichol to chime in on that.  Maybe if it is a top level config?